src/main/webapp/app/shared/constants/file-extensions.constants.ts

```diff
@@ -0,0 +1,25 @@
+export const FILE_EXTENSIONS = [
```

Comment:
Nitpick: Could you maybe sort the elements in this list and in `FileResource.java` in the same order? That would make a quick comparison easier.

Reply:
Done, thanks!